docusaurus/docs/how-to-guides/app-plugins/implement-rbac-in-app-plugins.md

```diff
@@ -65,6 +85,116 @@ For example, in the above example, users with the `Viewer` role will automatical
 
 When defining roles, ensure each role is clearly differentiated with unique permissions to avoid conflicts and unintended access. It's best to follow the principle of least privilege, assigning the minimum permissions necessary for the tasks.
 
+## Actionsets, extend actions with permissions
+
+The action set system provides a clean way for plugins to extend Grafana's existing permission model without requiring users to understand or manually configure individual plugin actions. 
+Action Sets are a powerful feature that allows you to define **named collections of actions** that can be referenced as a single unit and used to extend existing roles. 
+
+### Grafana Action Sets
+
+Grafana has several action sets for core resources that can be extended:
+
+**Folder Action Sets:**
+- `folders:view` → `["folders:read", "dashboards:read"]`
+- `folders:edit` → `["folders:read", "folders:write", "dashboards:read", "dashboards:write", "folders:create"]`
+- `folders:admin` → `["folders:read", "folders:write", "folders:delete", "folders.permissions:read", "folders.permissions:write", ...]`
+
+**Dashboard Action Sets:**
+- `dashboards:view` → `["dashboards:read"]`
+- `dashboards:edit` → `["dashboards:read", "dashboards:write"]`
+- `dashboards:admin` → `["dashboards:read", "dashboards:write", "dashboards:delete", "dashboards.permissions:read", ...]`
+
+### Limitations
+
+- **Folder/Dashboard Only**: Currently restricted to `folders:view/edit/admin`, and `dashboards:view/edit/admin` action sets
+- **No Scope Restriction**: Plugin permissions in action sets can't have scopes  
```

Comment:
IMO it would be helpful to describe the concept of scopes here (or point elsewhere even) as it wasn't immediately clear what this meant without seeing the example using scopes later in the doc.